docs/UserGuide.md

```diff
@@ -347,11 +353,14 @@ Format: `list`
 
 ![list](images/listCommand.png)
 
-### Listing all macros : `list`
+### Listing previously entered valid commands
```

Comment:
I'm just thinking is "Listing" the correct word to use since we are not truly "Listing" out the commands...
The actual feature is to trace back the commands one at a time, so maybe something like "access historical command" might be less misleading.

Reply:
Ohh yes this is a neat suggestion!